addons/account_peppol/tests/test_peppol_participant.py

```diff
@@ -106,17 +134,59 @@ def test_create_reject_participant(self):
     def test_create_duplicate_participant(self):
         # should not be possible to create a duplicate participant
         settings = self.env['res.config.settings'].create({})
-        settings.write({
-            'is_account_peppol_participant': True,
-            'account_peppol_eas': '9925',
-            'account_peppol_endpoint': '0000000000',
-            'account_peppol_phone_number': '+32483123456',
-        })
-
+        settings.write(self._get_participant_vals())
         settings.execute()
 
         with self._patch_peppol_requests():
             settings.button_create_peppol_proxy_user()
             with self.assertRaises(IntegrityError), self.cr.savepoint():
                 settings.account_peppol_proxy_state = 'not_registered'
                 settings.button_create_peppol_proxy_user()
+
+    def test_save_migration_key(self):
+        # migration key should be saved
+        settings = self.env['res.config.settings'].create({})
+        vals = self._get_participant_vals()
+        vals['account_peppol_migration_key'] = 'helloo'
+        settings.write(vals)
+        settings.execute()
```

Comment:
```suggestion
settings = self.env['res.config.settings']\
    .create({
        **self._get_participant_vals(),
        'account_peppol_migration_key': 'helloo'
    })\
    .execute()
```
same for all creation of settings